linux-tkg-config/prepare

```diff
@@ -506,23 +506,24 @@ _tkg_srcprep() {
     cd ${wrksrc}/linux-${_kern_ver}
   fi
 
-  
-  if [ -z "${_configfile}" ]; then    
+
+  if [ -z "${_configfile}" ]; then
     if [ "${_distro}" = "Arch" ] || [ "$_distro" = "Void" ]; then
       cat "${srcdir}"/config.x86_64 > ./.config
     else
       if [ -f /boot/config-`uname -r` ];then
-        _msg="Using /boot/config-`uname -r` as config file"
+        msg2 "Using /boot/config-`uname -r` as config file"
```

Comment:
Please respect the switch to the _msg variable for patch application messages :frog:

Reply:
That's the thing, it's not a patch, just an informative message :thinking:

Reply:
It was changed to allow for showing the messages dynamically:
https://github.com/Frogging-Family/linux-tkg/commit/09b5e06b5cee80a9a25ef71cd64ed2774cd14009

Reply:
Oh I see what you mean. I should go to sleep.

Reply:
Yes, and as I understand things `_tkg_patcher` needs to be called for the message to be displayed. It's not called here and `msg2` gets overridden after. For sure I don't understand something, I will look deeper into it.